cli/src/main.c

```diff
@@ -185,6 +189,29 @@ static int cmd_get_keys(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_delkeys(const struct shell *shell, size_t argc, char **argv){
+        ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+        printf("Keys deleted\n");
+        keys_counter = 0;
```

Comment:
Even if this memory is only accesible from the secure world, we should safely delete all memory allocated to the private and public keys

Reply:
Solved in new commit